JotunnLib/Managers/MockSystem/MockManager.cs

```diff
@@ -142,22 +147,40 @@ public static Object GetRealPrefabFromMock(Object unityObject, Type mockObjectTy
             }
 
             var unityObjectName = unityObject.name;
-            if (IsMockName(unityObjectName, out unityObjectName))
+            if (IsMockName(unityObjectName, out unityObjectName, out string pathName))
             {
-                // Cut off the suffix in the name to correctly query the original material
-                if (unityObject is Material)
+                unityObjectName = GetCleanedName(unityObject.GetType(), unityObjectName);
+
+                if (pathName != null)
```

Comment:
```suggestion
                if (!string.IsNullOrEmpty(pathName))
```
This is a bit cleaner, as an empty string cannot be matched as well